actors/builtin/market/set_multimap.go

```diff
@@ -14,23 +14,26 @@ import (
 )
 
 type SetMultimap struct {
-	mp    *adt.Map
-	store adt.Store
+	mp            *adt.Map
+	store         adt.Store
+	innerBitwidth int
 }
 
 // Interprets a store as a HAMT-based map of HAMT-based sets with root `r`.
-func AsSetMultimap(s adt.Store, r cid.Cid) (*SetMultimap, error) {
-	m, err := adt.AsMap(s, r)
+// Both inner and outer HAMTs are interpreted with branching factor 2^bitwidth.
```

Comment:
In general these two should be allowed to differ.  I think it makes sense to enable that in this PR with another parameter

Reply:
Ok